scss/example-theme/_overrides.scss

```diff
@@ -0,0 +1,2 @@
+// Use this file to define style overrides for @edx/paragon
```

Comment:
Do we want to include this here?  Or encourage people to make a brand repo?  Isn't adding code here sorta the 'wrong' way to do it, since they're effectively editing the paragon repository instead of creating a brand to alias?